scripts/onboarding/aks/onboarding-msi-bicep-syslog/existingClusterOnboarding.bicep

```diff
@@ -0,0 +1,67 @@
+@description('AKS Cluster Resource ID')
```

Comment:
what's preventing to merge both onboarding-msi-bicep-syslog and onboarding-msi-bicep bicep  to single bicep templates?

Reply:
Bicep does not support the traditional if-else construct where you can switch between two resources based on a condition. need to use two separate if statements for that, and that will introduce many if conditions in the template